extensions/amp-story/1.0/amp-story.js

```diff
@@ -344,6 +346,21 @@ export class AmpStory extends AMP.BaseElement {
         createPseudoLocale(LocalizedStringsEn, (s) => `[${s} one two]`)
       );
 
+    // If there are inline strings, register as current document language.
+    const inlineStrings = this.win.document.querySelector(
+      'script[amp-strings="amp-story"]'
```

Comment:
Is it still time to use a more specific word than just `string`? Would be cool to add `localization` or `i18n` or anything in the mix

Reply:
We can change it since the transformer wasn't merged AFAIK. Sth like what's below looks good to you?
```html
<script amp-localization="amp-story" i-amphtml-version="123456"></script>
```
cc @erwinmombay for the transformer change.